tests/test-default-pool/01-asset.yaml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
What is this in here for?

Reply:
How did that get left in there... Removed.